Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'rspec'
```

Comment:
Are the `rspec` and `rails-controller-testing` gems necessary?

Reply:
I was using the rails-controller-testing gem to test with render_template method, but since we'll add system testing maybe I can delete the gem as well as the test since it will be tested with more depth with the new tests.
What do you think?